docs/source/concepts/architecture/tests/test-sectors.rst

```diff
@@ -0,0 +1,59 @@
+Test Sectors for GeoIPS
+=======================
+
+We use small, pre-determined test sectors for testing. 
+These sectors provide a variety of scene components 
+including coast and land, edge of geostationary disk, day/night/terminator features etc. 
+
+Naming Convention
+-----------------
+
+Test sector plugin names follow this format:
+
+.. code-block::
+
+   test_<satellite>_<proj>_<res>_<edge|nadir>_<day|night|term>_<OPTIONAL:tc|ar|volc>_<YYYYMMDDTHHMMZ>.yaml
+
+Example
+~~~~~~~
+
+.. code-block::
+
+   test_goes16_eqc_3km_edge_day_tc_20200918T1950Z
+
+Components
+~~~~~~~~~~
+
+* **satellite**: The satellite platform (e.g., goes16, himawari8)
+* **proj**: Projection type (e.g., eqc for Equidistant Cylindrical (Plate Carrée) projection)
+* **res**: Resolution (e.g., 3km)
+* **edge|nadir**: Satellite view position
```

Comment:
I think edge/nadir should also be optional,  if it is not specifically edge of scan or directly nadir, just leave this field out